Dockerfile.dev

```diff
@@ -31,7 +31,8 @@ ENV AIRFLOW_HOME=/app \
     # AIRFLOW_SMTP_HOST= \
     # AIRFLOW_SMTP_USER= \
     # AIRFLOW_SMTP_PASSWORD= \
-    AIRFLOW_SMTP_FROM=telemetry-alerts@airflow.dev.mozaws.net
+    AIRFLOW_SMTP_FROM=telemetry-alerts@airflow.dev.mozaws.net \
+    AIRFLOW__SCHEDULER__CATCHUP_BY_DEFAULT=False
```

Comment:
It might be worth adding a comment linking to https://airflow.apache.org/configuration.html above this ENV directive for easy reference of how these env vars are used, especially since we're now using one of the more complex double-underscore overrides. In particular, it's nice to see docs about env vars taking precedence over config values.